src/module.ts

```diff
@@ -38,8 +38,13 @@ export class PrometheusModule {
         ),
       });
     }
+    providers.push({
+      provide: PROM_CONFIG,
+      useValue: {prefix: options?.prefix || ''},
+    })
 
     return {
+      global: true,
```

Comment:
delete? Not sure why this was added

Reply:
The global is added to inject the prefix into all providers throughout the service.

Reply:
The consumer can also import the `PrometheusModule` - which is something they need to do already anyway in order to inject metrics for other modules. So lets drop the `global` since it aligns with current behavior.